pkg/parser/parser.peg

```diff
@@ -717,6 +718,7 @@ LabeledListItemTermElement <- element:(Word
         / ConcealedIndexTerm
         / IndexTerm
         / InlinePassthrough
+        / InlineIcon
```

Comment:
could you add a few tests just to make sure that inline icons/images can be parsed in labeled list terms?

Reply:
I'm doing this .. so they only work in labeled list *descriptions*.

The grammar does not permit any markup in  the *terms*.  This should probably be considered a defect, but not one that I'm going to fix right now.

Reply:
(It's a preexisting defect.)